src/main/java/com/razorpay/ApiUtils.java

```diff
@@ -24,7 +25,10 @@
 
   static void createHttpClientInstance(boolean enableLogging) throws RazorpayException {
     if (client == null) {
-      client = new OkHttpClient.Builder().build();
+      client = new OkHttpClient.Builder()
+              .readTimeout(60, TimeUnit.SECONDS)
+              .writeTimeout(60, TimeUnit.SECONDS)
```

Comment:
@riagarg5 We should only care about read and write timeouts because we want to increase the response timeout for an API request (after the connection is already established)